domain-services/fiscal-event-service/src/main/resources/application.properties

```diff
@@ -43,4 +43,6 @@ ifix.master.government.search.path=/government/v1/_search
 
 ifix.master.project.host=http://localhost:8030
 ifix.master.project.context.path=/ifix-master-data
-ifix.master.project.search.path=/project/v1/_search
\ No newline at end of file
+ifix.master.project.search.path=/project/v1/_search
+
+mongock.change-logs-scan-package=org.egov.migration
```

Comment:
Add extra line break

Reply:
done

Reply:
Add the extra line break at the END of the file. Not here.